micro-airtable-api.test.js

```diff
@@ -0,0 +1,77 @@
+const http = require('http');
+const got = require('got');
+const listen = require('test-listen');
+const handler = require('./index');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+const baseConfig = {
+  airtableBaseId: process.env.AIRTABLE_BASE_ID,
+  airtableApiKey: process.env.AIRTABLE_API_KEY,
+};
+
+const getExistingPosts = async (client) => {
+  return (await client.get('/v0/Posts', {json: true})).body;
+};
+
+describeIfEnv('micro-airtable-api', () => {
+
+  describe('when `allowedMethods="*"`', () => {
+    const config = {...baseConfig, allowedMethods: '*'};
+    let server;
+    let client;
+
+    beforeAll(async () => {
+      server = http.createServer(handler(config));
+      const baseUrl = await listen(server);
+      client = got.extend({ baseUrl });
```

Comment:
If we set `json: true` here, it'll apply for all future calls too. This might be a good default to set since we know Airtable uses JSON exclusively.